tests/dataset/test_tabledoc.py

```diff
@@ -0,0 +1,81 @@
+"""Test the dataset module."""
+
+import pytest
+
+
+# if True:
+def test_as_dicts():
+    """Test the as_dicts() method."""
+
+    pytest.importorskip("rdflib")
+
+    from tripper import DCAT, EMMO, Namespace, Triplestore
+    from tripper.dataset import TableDoc
+
+    ONTO = Namespace("http:/example.com/onto#")
+    DS = Namespace("http:/example.com/datasets#")
+
+    td = TableDoc(
+        header=[
+            "@id",
+            "@type",
+            "@type",
+            "inSeries",
+            "distribution.downloadURL",
+        ],
+        data=[
+            ("ds:s1", "onto:T1", "onto:T2", None, "file:///data/"),
+            ("ds:d1", "onto:T1", None, "ds:s1", "file:///data/d1.txt"),
+            ("ds:d2", "onto:T2", None, "ds:s1", "file:///data/d2.txt"),
+        ],
+        prefixes={
+            "onto": "http:/example.com/onto#",
+            "ds": "http:/example.com/datasets#",
+        },
+        # context={
+        #    "ds": "http:/example.com/datasets#",
+        # },
```

Comment:
I would like to keep this comment, since the intention is to uncomment it when we have a fully working implementation of the `context` keyword argument. Added a comment to clarify this